app/services/content_change_deduplicator_service.rb

```diff
@@ -0,0 +1,18 @@
+class ContentChangeDeduplicatorService
+  def self.call(*args)
+    new.call(*args)
+  end
+
+  def call(content_changes)
+    deduplicated = reverse_by_public_updated_at(content_changes)
+      .uniq(&:content_id)
+
+    deduplicated.sort_by(&:public_updated_at)
```

Comment:
Should we return the results ordered alphabetically?

Reply:
No 🤣